ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/offers/SettlementMapper.java

```diff
@@ -38,33 +38,34 @@ public Settlement fromCCDDefendant(CCDDefendant ccdDefendant) {
         if (CollectionUtils.isEmpty(ccdDefendant.getSettlementPartyStatements())) {
             return null;
         }
-        List<PartyStatement> partyStatements = ccdDefendant.getSettlementPartyStatements().stream()
-            .map(partyStatement -> partyStatementMapper.from(partyStatement))
-            .collect(Collectors.toList());
+
 
         Settlement settlement = new Settlement();
-        partyStatements.forEach(partyStatement -> addPartyStatement(partyStatement, settlement));
+        ccdDefendant.getSettlementPartyStatements().stream()
+            .map(partyStatementMapper::from)
+            .forEach(partyStatement -> addPartyStatement(partyStatement, settlement));
 
         return settlement;
     }
 
     private void addPartyStatement(PartyStatement partyStatement, Settlement settlement) {
-        if (partyStatement.getType().equals(StatementType.OFFER)) {
-            settlement.makeOffer(partyStatement.getOffer().orElse(null),
+        if (StatementType.OFFER.equals(partyStatement.getType())) {
+            settlement.addOffer(
```

Comment:
The only functional change in this file is here

Reply:
I agree with you it doesn't need, same thing can be done in `formaliseSettlement`  method of `FormaliseResponseAcceptanceService` as well. However was it effecting anything or any failures notices

Reply:
I'm not familiar with that code, but isn't it actually making the offers and so would need the validation?

Reply:
This change looks fine here.